testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorTest.java

```diff
@@ -5909,6 +5913,234 @@ private void testMaterializedViewColumnName(String columnName, boolean delimited
         assertUpdate("DROP MATERIALIZED VIEW " + viewName);
     }
 
+    @Test
+    public void testProjectionPushdown()
+    {
+        String tableName = "test_projection_pushdown_" + randomNameSuffix();
+
+        if (hasBehavior(SUPPORTS_CREATE_TABLE_WITH_DATA) && hasBehavior(SUPPORTS_ROW_TYPE)) {
+            assertUpdate("CREATE TABLE " + tableName + " (id BIGINT, root ROW(f1 BIGINT, f2 BIGINT))");
```

Comment:
use `TestTable` to manage tables.

some connector tests run against shared resource, so we should ensure test tables are cleaned up even if the test fails.